### ISSUE TITLE
Support negative prompt

### ISSUE THREAD

**Issue by realisticdreamer114514** [Haoming02/sd-forge-couple#59]
I said it here [#58](https://github.com/Haoming02/sd-forge-couple/issues/58#issuecomment-2343471034) it doesn't work on negative prompt yet but regional prompter does, it goes a long way if I can eliminate things I don't want to see in each layer

**Comment by Haoming02** [comment]
Well, people are already having enough difficulty understanding the current implementation *(despite the existence of explicitly written README)*, imagine the complexity when you add yet aspect to the mix.

Also, to add this function, you need to completely redesign: **1.** frontend UI; **2.** frontend code; **3.** backend code, for all 3 modes. That's already 9 parts. And once this is added, some people definitely would like a toggle for it *(**eg.** me, so far I haven't had the need for regionalized negative prompt)*. So that's ***18*** major changes...

## TL;DR

Nah

**Comment by Reithan** [comment]
It's been a while - any possibility of supporting this yet?
